### PR TITLE
Mergeback 1.4.2rc5

### PR DESCRIPTION
**Pull request by yunchu**
### Summary

Mergeback PR for 1.4.2rc5

@sovrasov could you review this carefully for applying following changes into develop?
https://github.com/openvinotoolkit/training_extensions/pull/2483
https://github.com/openvinotoolkit/training_extensions/pull/2459

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: src/otx/api/usecases/exportable_code/demo/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 openvino==2023.0
 openvino-model-api==0.1.5
-otx @ git+https://github.com/sovrasov/openvino_training_extensions/@38a9d7b20e3359d6139616c7127eac98d304c9f6#egg=otx
+otx @ git+https://github.com/openvinotoolkit/training_extensions/@e066a04834952257c1c9384a09f472d13b76b264#egg=otx
```
Comment (goodsong81) [inline src/otx/api/usecases/exportable_code/demo/requirements.txt:3]: Is this commit OK?

**Review by goodsong81 — APPROVED** [review]
LGTM. Just 1 question below. Thanks!

File: src/otx/api/usecases/exportable_code/demo/requirements.txt (hunk 1 above)
Comment (yunchu) [reply src/otx/api/usecases/exportable_code/demo/requirements.txt:3]: this is the current develop head. so I think it's okay. is there any concern?

Comment (goodsong81) [reply src/otx/api/usecases/exportable_code/demo/requirements.txt:3]: Current head is 6f00907ec67b4d601601e71e790109673ca56d82.
And it does not need to be the develop head but the latest commit in this PR if there is some changes in `otx.api` in this PR.

**Review by sovrasov — COMMENTED** [review]
LGTM from my side

**Review by sovrasov — APPROVED** [review]

**Pull request merged**